apps/sports-helsinki/src/constants.ts

```diff
@@ -2,9 +2,15 @@ import type { Language } from './types';
 
 export const DEFAULT_LANGUAGE = 'fi';
 
-export const ROUTES = {
+export const SEARCH_ROUTES = {
   SEARCH: '/search',
+  MAPSEARCH: '/search/map',
   COURSESEARCH: '/coursesearch',
+};
+
+// TODO: Set a typing that tells that it is extending SEARCH_ROUTES
+export const ROUTES = {
+  ...SEARCH_ROUTES,
```

Comment:
The way how the routes were handled was a bad one and there was also a bug. I refactored it, but the way is still not ideal. It would be nice to type the search routes and routes so that it’s clear that the routes extends the search routes and the possible routes for the search commands are only the search routes.

The SEARCH_ROUTES really are included in the ROUTES, but the ROUTES type does not tell that:
```
export const SEARCH_ROUTES = {
  SEARCH: '/search',
  MAPSEARCH: '/search/map',
  COURSESEARCH: '/coursesearch',
};

// TODO: Set a typing that tells that it is extending SEARCH_ROUTES
export const ROUTES = {
  ...SEARCH_ROUTES,
  COURSES: '/courses/[eventId]',
  VENUES: '/venues/[venueId]',
  ARTICLE_ARCHIVE: '/article-archive',
  ARTICLES: '/articles/[...slug]',
  PAGES: '/pages/[...slug]',
  LINK: '',
};
```
forwardPath should be 1 of the SEARCH_ROUTES. Now its typed just as string:

```
export const extractLatestReturnPath = (
  queryString: string,
  locale: string,
  forwardPath: typeof SEARCH_ROUTES[keyof typeof SEARCH_ROUTES] = SEARCH_ROUTES.SEARCH // TODO: Allow only SEARCH_ROUTE values
): ReturnParams => {
```